### PR TITLE
74 verifyelementtext doesnt highlight the elementtext under test when running the test case

### LINKED ISSUE #74: VerifyElementText doesn't highlight the elementtext under test when running the test case
When running test with keyword VerifyElementText it doesn't highlight the element text under test. It would be improvement to see and confirm that the keyword tests certain element text with anchor (because the same text can be multiple times in different places on the page).  

Example:
`VerifyElementText    ipsum    ipsum    5    dolore`

`    SetConfig    SearchMode           Draw`
Doesn't work either with VerifyElementText

### PR DESCRIPTION
**Pull request by turunenm**
Includes a fix and a new acceptance test in _element.robot_
Placed the test under _element.robot_ since this tests the `draw border feature` and not the `VerifyElementText`

### CODE REVIEW

File: test/acceptance/element.robot [hunk 1]
```diff
@@ -247,4 +246,23 @@ VerifyAttributeNOK
...
     # multiple found css
-    Run Keyword And Expect Error      QWebValueError:*              VerifyAttribute      button               value                Button2              element_type=css     timeout=2
\ No newline at end of file
+    Run Keyword And Expect Error      QWebValueError:*              VerifyAttribute      button               value                Button2              element_type=css     timeout=2
+
+VerifyElementTextBorders
+    [Tags]                      VerifyElementText    DrawBorders
+    Go To                       file://${CURDIR}/../resources/text.html
+    
+    ${no_border_file}           SetVariable            ${CURDIR}/no_border.png
```
Comment (tkoukkari) [inline test/acceptance/element.robot:255]: I can't see these images, should they be included?

Also maybe could just use VerifyIcon here?

Comment (turunenm) [reply test/acceptance/element.robot:255]: Those are just filepath strings for screenshots which will be taken during the the Test and there should be no need for stored images

Comment (turunenm) [reply test/acceptance/element.robot:255]: To clarify, test takes screenshot from initial state with no element highlights.
Then an element is highlighted and a new screenshot is taken.
The screenshots are then compared on binary level and they should be different.

VerifyIcon would also work but requires an existing image.

I don't actually know which would be better approach 😅

Comment (tkoukkari) [reply test/acceptance/element.robot:255]: ah, okay. That explains why I couldn't find those images :).

**Review by tkoukkari — APPROVED** [review]
lgtm

**Review by Ekexyz — APPROVED** [review]

**Pull request merged**